tests/10apidoc/03events-initial.pl

```diff
@@ -142,7 +142,7 @@ sub await_event_for
          my $found;
          foreach my $event ( @_ ) {
             not $found and $filter->( $event ) and
```

Comment:
Cute though the terseness of this loop is, perhaps it could be expanded into something more legible?